packages/elements/src/components/ino-markdown-editor/ino-markdown-editor.tsx

```diff
@@ -307,7 +307,6 @@ export class MarkdownEditor implements ComponentInterface {
           </div>
           <ino-popover
             placement="top-start"
-            color-scheme="transparent"
```

Comment:
Why you removed this? Only `'secondary' & 'light'` was removed from the `ino-popover`